docs/configuration.md

```diff
@@ -0,0 +1,393 @@
+# Configuration
+
+Bee uses the Toml standard as config file. If you are unsure about some syntax have a look at the official specs [here](https://toml.io).
+The default config file is `config.toml`. You can change the path or name of the config file by using the `-c` or `--config` flag. 
+For Example: `bee -c config_example.toml`
+
+## Table of content
+
+[//]: # "Table of contents created with: http://ecotrust-canada.github.io/markdown-toc/"
+
+- [1. Logger](#1-logger)
+  * [outputs](#outputs)
+- [2. Network](#2-network)
+    + [peering](#peering)
+- [3. Protocol](#3-protocol)
+  * [coordinator](#coordinator)
+    + [public_key_ranges](#public-key-ranges)
+  * [workers](#workers)
+- [4. Rest_api](#4-rest-api)
+- [5. Snapshot](#5-snapshot)
+- [6. Pruning](#6-pruning)
+- [7. Storage](#7-storage)
+  * [storage](#storage)
+  * [env](#env)
+- [8. Tangle](#8-tangle)
+- [9. Mqtt](#9-mqtt)
+- [10. Dashboard](#10-dashboard)
+  * [auth](#auth)
+
+---
+
+| Name       | Description                                                                                | Type   |
+| :--------- | :----------------------------------------------------------------------------------------- | :----- |
+| identity   | hex representation of an Ed25519 keypair. Can be generated with the `bee p2pidentity` tool | string |
+| alias      | alias for your node. Shows up in dashboard                                                 | string |
+| bech32_hrp | network address identifier                                                                 | string |
+| network_id | network identifier                                                                         | string |
+
+## 1. Logger
```

Comment:
I wouldn't number the categories, doesn't really add anything and makes it annoying if/when we reorder them

Reply:
You are right. Problem with that is always, that sometimes you don't have a unique identifier anymore. For example with `[storage]` and `[storage.storage]`

Reply:
That's just an issue for the anchor, right ? Do we really care since it's not shown ?

Reply:
The problem is, that in the ToC I need to link to the storage table and the other storage table in the storage table. How should I do that, if the both have the same name? Or do I don't get something?